### DISCUSSION TITLE
codespace

### DISCUSSION THREAD

**Discussion by MoShaco** [community/community#29503] [Other Feature Feedback, Questions, & Ideas]
hello every one I need some help :)
I can not connect with my codespace, I do not know.
![GITHUP](https://user-images.githubusercontent.com/106908833/184612355-57a5af73-3b3b-413d-ab1c-5dad5c3e0a69.jpg)

**Comment by softwaredevelop** [comment]
try with [VS Code](https://code.visualstudio.com/) + [GitHub Codespaces extension](https://marketplace.visualstudio.com/items?itemName=GitHub.codespaces)
